ledger-core/v2/ledger/object/record_db.go

```diff
@@ -10,20 +10,18 @@ import (
 	"context"
 	"encoding/binary"
 	"fmt"
-	"sync"
 
 	"github.com/dgraph-io/badger"
 	"github.com/pkg/errors"
 
 	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
-	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/record"
 	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/store"
 	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
 )
 
 // RecordDB is a DB storage implementation. It saves records to disk and does not allow removal.
 type RecordDB struct {
-	batchLock sync.Mutex
+	// batchLock sync.Mutex
```

Comment:
ditto - there is no need to keep this line here.

Reply:
this gonna be removed soon